src/main.js

```diff
@@ -1,10 +1,23 @@
 const game = new Game();
+const player = new Player();
+let img;
+let treasure;
+const gold = new Treasure();
+
+function preLoad() {
+  treasure = loadImage("./assets/treasure.png");
+  img = loadImage('./assets/character-down.png');
+  game.preload();
+}
```

Comment:
Your `main.js` should only contain code used to setup the game and call the necessary p5.js to setup the canvas.

That being said:
1) Remove line 2 - you are already creating a player inside `Game`'s constructor (like you should!)
2) To organize the code a bit, declare `img` & `treasure` in the `constants.js`. Might be wise to give `img` a more descriptive name too, like `playerImg` for example :v:
3) Remove line 5. `Treasure` should be a part of the game, like `Player`.
4) Remove lines 8 & 9. `game.preload()` will call the `player.preload()` & `treasure.preload()`, which will take care of preloading the images.